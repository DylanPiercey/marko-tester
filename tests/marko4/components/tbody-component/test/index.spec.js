const { render, fixtures } = require('../../../../../src/index')('../index.marko');

describe('When component is rendered', () => {
  let component;

  beforeEach(() => {
    component = render(fixtures.default);
  });

  afterEach(() => {
    component.destroy();
  });

  it('should be able to access the DOM correctly', () => {
    expect(/^<tbody>/.test(document.body.innerHTML)).toBe(true);
  });
});
