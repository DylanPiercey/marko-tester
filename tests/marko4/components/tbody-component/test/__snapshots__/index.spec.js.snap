// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component with default fixture 1`] = `
Array [
  ,
  <tbody>
    <tr>
      <td>
        tbody-component 
        <lasso-img
          src="."
        />
      </td>
    </tr>
  </tbody>,
  ,
]
`;
